scheduler/scheduler.go

```diff
@@ -245,7 +250,8 @@ func createResources(toCreate chan *ScheduledResource, created chan string, ccLi
 			}
 
 			for {
-				time.Sleep(time.Millisecond * 100)
+				log.Printf("Checking status for %s", r.Key())
+				time.Sleep(time.Millisecond * 1000)
```

Comment:
Hmmm... Quiet big change... IMO this time should be a constant and should be later reused in https://github.com/Mirantis/k8s-AppController/pull/52/files#diff-9667cfa1ad6b8b445794f7c2469f069fR273

Maybe instead of informing about every check - inform below that status is changed to ready?